### DISCUSSION TITLE
📋🌟 create a list or projects or update the existing one to show last release, commit #, last commit, stars,

### DISCUSSION THREAD

**Discussion by ghost** [OWASP/www-committee-project#44] [Ideas]
see this as a starting point https://github.com/psiinon/owasp-projects 

we can create a github action that runs daily to update the list
make sure it shows - 
primary language, license, Forks, stars, Issues, PRs, issues need help last updated, Release version, date released  
figure out a way to show project statuses from each github repo to link roadmap issues similar to how the github list of repos has the stats

**Comment by ghost** [comment]
Found this https://github.com/OWASP/OWASP-Project-Metrics

**Comment by bkimminich** [comment]
I'm all for adding more information, only the GitHub README layout will not survive more columns, see https://github.com/bkimminich/owasp-projects - but of course as a test ground that's fine and could then later become an actual static website. Might even use a tile layout instead of table layout on smaller screens then.

**Comment by ghost** [comment]
it could be a csv too those are auto formatted by Github - ideally it would be sortable too - we could also have a generated html table thanks for the updated version
